tests/test_views.py

```diff
@@ -3509,6 +3509,14 @@ def test_folder_new_post(self):
                 found_item = True
         assert_true(found_item, "Did not find the folder in the dashboard.")
 
+    def test_update_user_timezone_offset(self):
+        assert_equal(self.creator.timezone, 'Etc/UTC')
+        payload = {'timezone': 'America/New_York'}
+        url = api_url_for('update_user', uid=self.creator._id)
+        self.app.put_json(url, payload, auth=self.creator.auth)
+        self.creator.reload()
+        assert_equal(self.creator.timezone, 'America/New_York')
+
```

Comment:
Yeah, I think the test will go in the test_notifications.py as a test of the utility function.